scripts/ci/forbidden-identifiers.js

```diff
@@ -33,7 +34,16 @@ const blocked_statements = [
   'from \\\'rxjs/Rx\\\'',
 ];
 
+// Retrieves all scope packages dynamically from the source.
+// Developers shouldn't be able to import from other scope packages by using relative paths.
+const componentFolders = fs
+  .readdirSync(`${__dirname}/../../src/components`)
+  .map(componentName => `src/components/${componentName}`);
+
+const scopePackages = ['src/core'].concat(componentFolders);
+
 const blockedRegex = new RegExp(blocked_statements.join('|'), 'g');
+const importRegex = /from '(.*)'/g;
```

Comment:
This doesn't check if the line is an `import`. What if `from '...'` is put inside a comment?